src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -261,4 +266,23 @@ private SpeechletResponse askForCity(Optional<String> currentCityName) {
         }
         return SpeechletResponse.newAskResponse(out, cityReprompt);
     }
+
+    private String allRegionsSpoken() {
+        List<String> activeRegions = obaClient.getAllRegions()
+                .stream()
+                .filter(r -> RegionUtils.isRegionUsable(r) && !r.getExperimental())
+                .map(r -> String.format("%s, ", r.getName()))
+                .collect(Collectors.toList());
```

Comment:
Can you sort these by alpha on region name as well?  I believe the current sorting is just by `regionId`, which seems arbitrary to the user.